Source/JTSImageViewController.m

```diff
@@ -590,6 +590,15 @@ - (void)setupDetailViews {
     self.detailLabel.translatesAutoresizingMaskIntoConstraints = false;
     [self.view addSubview:self.detailLabel];
 
+    if (!self.imageInfo.detailText && !self.imageInfo.timeText && !self.imageInfo.dateText && !self.imageInfo.title) {
```

Comment:
many blobs have everything except for detailText (which is our annotationText). and just because the blob doesn't have annotationText this is going to hide everything.